omero/users/history.rst

```diff
@@ -26,12 +26,46 @@ omero clients:
 - omero-web :web_source:`CHANGELOG.md<CHANGELOG.md>`
 
 .. Content for this page should be opened as a PR against
-.. https://github.com/ome/openmicroscopy/blob/develop/history.rst
+.. https://github.com/openmicroscopy/openmicroscopy/blob/develop/history.rst
```

Comment:
This change doesn't seem right.

Reply:
Maybe from the template file. I will need to check that